Source/Protocols/Request.swift

```diff
@@ -19,3 +21,9 @@ public extension Request where ResponseType: CollectionType, ResponseType.Genera
     return .fromDecoded(ResponseType.decode(j))
   }
 }
+
+public extension Request where ResponseType == EmptyResponse {
+  func parse(j: JSON) -> Result<ResponseType, NSError> {
+    return .Success()
```

Comment:
This is what our implementation looks like too, but is it too naive? I guess we wouldn't get to this point if there were errors along the way?